.travis.yml

```diff
@@ -0,0 +1,8 @@
+dist: xenial   # required for Python >= 3.7
+language: python
+python:
+  - "3.7"
+install:
+  - pip install pylint
+# command to run tests
+script: pylint osbuild
```

Comment:
Does this mean to only test the `osbuild` module? (I guess that's fine, given that we'll be moving away from the `osbuild` script soon.)